test/test-77-compare-dicts-and-tests/main.js

```diff
@@ -6,18 +6,9 @@ const fs = require('fs');
 const path = require('path');
 const assert = require('assert');
 
-let configs = fs.readdirSync('../../dictionary');
-configs = configs
-  .filter(function (config) {
-    return config !== '.eslintrc.json';
-  })
-  .map(function (config) {
-    assert.strictEqual(config.slice(-3), '.js');
-    return config.slice(0, -3);
-  });
-
-let tests = fs.readdirSync('../test-79-npm');
-tests = tests.filter(function (test) {
+const configs = Object.keys(require('../../lib-es5/dictionary').default);
```

Comment:
presumably this all of the sudden needs to require the `lib-es5` version because it's the transpiled javascript, not the direct javascript it was previously, right?